### ISSUE TITLE
Error ITMS-90482 contains bitcode when pushing to Apple store

### ISSUE THREAD

**Issue by jindrichgorner** [bugfender/BugfenderSDK-iOS#81]
Hi,

Currently using a Bugfender 1.9.2 in our ionic/capacitor app. When we are pushing to the Apple store we are getting the following error:

```
 Please correct the following issues, then upload again.
ITMS-90482: Invalid Executable - The executable 'App.app/Frameworks/BugfenderSDK.framework/BugfenderSDK' contains bitcode.
```

The ENABLE_BITCODE flag is set to NO everywhere in our project. 

Any idea what could be wrong? 

Thank you.

**Comment by jgimenez** [comment]
Hi @jindrichgorner , some projects require Bitcode and others don't. I would recommend enabling Bitcode for your project, but if you prefer to have it without, you can strip Bitcode out of the Bugfender SDK.

You can use the [bitcode_strip](http://www.manpagez.com/man/1/bitcode_strip/) tool for that.

**Comment by jindrichgorner** [comment]
Hi @jgimenez  thanks a lot for the quick reply! We had BITCODE allowed for our project but then we had issues with some other packages so unfortunately we had to turn it off. 

Okay so if I keep the ENABLE_BITCODE to NO and then strip out the bitcode from the BugfenderSDK that should do the job I assume?

In terms of actually enabling it. So I actually tried to enable bitcode for BugfenderSDK Pod target but was still getting the error from Apple. Was this because my Pods project still had the bitcode disabled? So in order to enable it I would need to set Pods project to YES and Bugfender pods target to YES I assume?

Thank you.

**Comment by jindrichgorner** [comment]
We have stripped the bitcode and now the error goes from "contains bitcode" to "incomplete bitcode" (ITMS-90668).
I am not sure whether there might be an issue with the way I used the bitcode_strip. I noticed there are 3 folders (under Pods/BugFenderSDK/BugFenderSDK.xcframework) - ios-armv7_arm64, ios-i386_x86_64-simulator, ios-x86_64-maccatalyst. So I run the following for all three folders in Podfile 
<img width="973" alt="image" src="https://user-images.githubusercontent.com/89532773/209174110-f60620d3-1d5b-4471-94cb-aa567404bf26.png">

**Comment by jgimenez** [comment]
I'm not sure what could be going on, nor how this chunk of code fits in your larger pipeline to make the final build. Unfortunately, each app is quite unique.
If you'd like to provide a sample project that we can debug, or would like to share more specific details on how you're doing the build, please feel free to share them here or over chat with our customer support.

**Comment by jindrichgorner** [comment]
Hi, 

Thanks for the reply. The chunk of code is in Podifle run in the `post_install 
do |installer|` section. I am not sure whether that is a correct way to strip the bitcode of Bugfender. I have included the whole podfile.  [Podfile.txt](https://github.com/bugfender/BugfenderSDK-iOS/files/10344067/Podfile.txt)

We then archive the project and create IPA from it. The submission is then done by a different team than us ( and each submission whether succesful or failed costs us money hence we are careful with trial&error for submission).  They use Transporter to upload the IPA to App Store Connect.

We actually have a different project with the same package that got accepted successfully a few months back. The only difference seems to be XCode version used to archive the project. The previous project was done using XCode 13 while the current one is being done using XCode 14. Looking at the notes for XCode 14 it seems bitcode submissions are not accepted anymore? So I guess the only way is to strip the bitcode unless I am misunderstanding something regarding how Bitcode works?

**Comment by jgimenez** [comment]
Hi @jindrichgorner, unfortunately, we've not been able to reproduce the problem, so I'm inclined to think this is something related to your specific project configuration. If you could share a sample project that exhibits this behavior, we can give it a look. You can share it here or via our support channel.

**Comment by jgimenez** [comment]
I'm closing due to inactivity. Please let us know if you still need help and have more details on how to reproduce the problem.

**Comment by Aleksefo** [comment]
We have the same issue in our project. Happens after we updated some libraries and activated Hermes.

❌  ld: '/Users/[REDACTED]/Library/Developer/Xcode/DerivedData/[REDACTED]/Build/Intermediates.noindex/ArchiveIntermediates/freud/BuildProductsPath/Release-iphoneos/XCFrameworkIntermediates/BugfenderSDK/BugfenderSDK.framework/BugfenderSDK' does not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. file '/Users/[REDACTED]/Library/Developer/Xcode/DerivedData/[REDACTED]/Build/Intermediates.noindex/ArchiveIntermediates/freud/BuildProductsPath/Release-iphoneos/XCFrameworkIntermediates/BugfenderSDK/BugfenderSDK.framework/BugfenderSDK' for architecture arm64
❌  clang: error: linker command failed with exit code 1 (use -v to see invocation)

**Comment by jindrichgorner** [comment]
We managed to get past this issue by changing the flag in the strip_command (the code above) from -m to -r. This will completely strip Bugfender of any BITCODE.  We have also set ENABLE_BITCODE to NO everywhere in our project. These changes allowed us to do the submission to the store. 

This is our post_install in the Podfile; 

<img width="1327" alt="image" src="https://user-images.githubusercontent.com/89532773/222117019-7b84e211-3514-46f8-a94b-0578ddb53de4.png">
